### PR TITLE
feat: support generate list component

### PR DESCRIPTION
**Pull request by zrg-team**
Support List component

### CODE REVIEW

File: src/components/atoms/List/List.component.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import { PreviewProps } from '@/types/preview'
+import { ComponentProps } from '@/types/component'
+
+export interface ListProps<T> extends PreviewProps, ComponentProps<AntListProps<T>> {}
+
+const List = React.forwardRef<HTMLDivElement, ListProps<any>>((props, ref) => {
+  const { style, ...rest } = props
+  return (
+    <div style={style} ref={ref}>
+      <AntList style={{ width: '100%', height: 'auto' }} {...rest} />
```
Comment (quy-truong) [inline src/components/atoms/List/List.component.tsx:13]: should we use inline or class, we already support it now

File: src/components/atoms/List/List.component.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { List as AntList, ListProps as AntListProps } from 'antd'
+
+import { PreviewProps } from '@/types/preview'
+import { ComponentProps } from '@/types/component'
+
+export interface ListProps<T> extends PreviewProps, ComponentProps<AntListProps<T>> {}
+
+const List = React.forwardRef<HTMLDivElement, ListProps<any>>((props, ref) => {
```
Comment (quy-truong) [inline src/components/atoms/List/List.component.tsx:9]: T here?

**Review by quy-truong — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jitera/jitera-web-ui-library",
-  "version": "1.4.2",
+  "version": "1.4.3",
```
Comment (duongjitera) [inline package.json:3]: no need to change this, it acutally 1.7.x

**Review by duongjitera — APPROVED** [review]

**Review by jefrydco — APPROVED** [review]

File: src/components/atoms/List/List.component.tsx (hunk 1 above)
Comment (zrg-team) [reply src/components/atoms/List/List.component.tsx:13]: @quy-truong i will update now

File: src/components/atoms/List/List.component.tsx (hunk 2 above)
Comment (zrg-team) [reply src/components/atoms/List/List.component.tsx:9]: @quy-truong it because AntListProps but i will remove it too

File: package.json (hunk 3 above)
Comment (zrg-team) [reply package.json:3]: @duongjitera i will update

**Pull request merged**